### DISCUSSION TITLE
Can I use this to check XPath?

### DISCUSSION THREAD

**Discussion by mohsin** [upptime/upptime#332] [Ideas]
I'm trying to use this for a more specific use-case i.e. to check the availability of a product on a website. Now right now I see the way that this works is that it checks if the website is up and running and then reports the status. Can I configure this to use a XPath selector to test a string matching instead and then report if the string doesn't match?

**Comment by AnandChowdhary** [comment]
We don't have this feature right now, but I'm happy to accept a PR.
